### DISCUSSION TITLE
How do you develop the Compose ui without previews?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [Shabinder/SpotiFlyer#235] [Q&A]
Hey guys, seeing that the spotyFlyer has a rather robust ui and lots of composable functions for each screen, I was wondering how do you go about developing these screens without having preview annotation available in the KMP at the moment.

**Comment by Shabinder** [comment]
Good Question , 

To be honest in early days of compose adoption I tried to rerun the app on my device to check the output

But as of now , I am pretty confident while building UI's so I just straight forward go and code UI for a whole screen and then simply run , most of the times I get the desired output which I had in my mind and sometimes I achieve that by doing some simple modification after initial run.

However I read somewhere that Previews have come for KMP code too , I will soon integrate preview fuction for each component in app as soon as possible.
